code/datums/achievements/job_achievements.dm

```diff
@@ -2,14 +2,56 @@
 /datum/award/achievement/jobs
 	category = "Jobs"
 
-//chemistry
+//engineering
+
+/datum/award/achievement/jobs/theoretical_limits
+	name = "All Within Theoretical Limits"
+	desc = "I never thought I'd see a resonance cascade, let alone prevent one..."
+	database_id = MEDAL_THEORETICAL_LIMITS
+	icon = "theoreticallimits"
+
+//medical
+
+/datum/award/achievement/jobs/sandman
+	name = "Mister Sandman"
+	desc = "Mechanically speaking, there's no real benefit to being unconscious during surgery. Weird how insistent this doctor is about using the N2O anyway though, huh?"
+	database_id = MEDAL_SANDMAN
+	icon = "basemisc"
+
+/datum/award/achievement/jobs/helbitaljanken
+	name = "Helbitaljanken"
+	desc = "You janked hard"
+	database_id = MEDAL_HELBITALJANKEN
+	icon = "helbital"
 
 /datum/award/achievement/jobs/chemistry_tut
 	name = "Perfect chemistry blossom"
 	desc = "Passed the chemistry tutorial with perfect purity!"
 	database_id = MEDAL_CHEM_TUT
 	icon = "chem_tut"
 
+//mining
+
+/datum/award/achievement/jobs/frenching
```

Comment:
I'm not sure about this one. It involves doing something obscure, silly, completely unrelated to the job beside the fact it's Bubblegum.